src/utils/keychainConstants.js

```diff
@@ -0,0 +1,6 @@
+export const seedPhraseKey = 'rainbowSeedPhrase';
```

Comment:
can we update redux/wallets.js line 290: const seedKeyFound = await hasKey(`${key}_rainbowSeedPhrase`); to use seedPhraseKey